shell/platform/darwin/ios/BUILD.gn

```diff
@@ -37,15 +37,7 @@ _flutter_framework_headers = [
 
 _flutter_framework_headers_copy_dir = "$_flutter_framework_dir/Headers"
 
-shared_library("create_flutter_framework_dylib") {
-  visibility = [ ":*" ]
-
-  output_name = "Flutter"
-
-  ldflags = [ "-Wl,-install_name,@rpath/Flutter.framework/Flutter" ]
-
-  public = _flutter_framework_headers
-
+source_set("flutter_framework_source") {
   cflags_objc = flutter_cflags_objc
```

Comment:
we can add a non-public visibility clause here to be clean

Reply:
done